lmcache/storage_backend/serde/cachegen_basics.py

```diff
@@ -74,6 +73,30 @@ def from_model_name(model_name: str) -> "CacheGenConfig":
                 value_first_bins=32,
                 value_second_bins=16,
             )
+        elif model_name == "meta-llama/Llama-3.1-8B-Instruct":
```

Comment:
Seems like this does not belong to this PR?

Reply:
No